packages/xo-server/src/api/vtpm.mjs

```diff
@@ -0,0 +1,27 @@
+export function create({ vm }) {
+  return this.getXapi(vm).VTPM_create({ VM: vm._xapiRef })
+}
+
+create.description = 'create a VTPM'
+
+create.params = {
+  id: { type: 'string' },
+}
+
+create.resolve = {
+  vm: ['id', 'VM', 'administrate'],
+}
+
+export function destroy({ vtpm }) {
+  return this.call('VTPM.destroy', vtpm._xapiRef)
```

Comment:
You are not inside a `Xapi` but an `Xo`  instance, you cannot call `Xapi#call` directly.